ee/tabby-db/migrations/0022_github-provider.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE github_repository_provider(
+    id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    display_name TEXT NOT NULL,
+    application_id TEXT NOT NULL,
+    secret TEXT NOT NULL,
+    CONSTRAINT `idx_application_id` UNIQUE (`application_id`)
+);
+CREATE TABLE github_provided_repositories(
+    id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    provider_id INTEGER NOT NULL,
```

Comment:
should this be a foreign key to provider table?
if so, it should be `github_repository_provider_id`